resources/lib/monitor.py

```diff
@@ -312,7 +312,7 @@ def post_capabilities(self, server):
                 "Mute,Unmute,SetVolume,"
                 "Play,Playstate,PlayNext,PlayMediaSource"
             ),
-            'IconUrl': "https://raw.githubusercontent.com/MediaBrowser/plugin.video.emby/develop/kodi_icon.png",
+            'IconUrl': "https://raw.githubusercontent.com/MediaBrowser/plugin.video.jellyfin/develop/kodi_icon.png",
```

Comment:
WAT? :D

Reply:
Search and replace

Reply:
@cvium Can we change this URL to some of ours?